CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## master
 
+## [v5.3.0][v5.3.0] (february 4, 2022)
+
+* Added the [Fiber](https://github.com/gofiber/fiber) integration ([#227](https://github.com/airbrake/gobrake/pull/227)),
+([#232](https://github.com/airbrake/gobrake/pull/232))
+
+* Minor bug fixes ([#227](https://github.com/airbrake/gobrake/pull/228))
```

Comment:
Don't include `minor bug fixes` because it's not very descriptive. Either delete this entirely, or specify what bug fixes were done.